TechnicalDocuments/SnapshotTesting.md

```diff
@@ -0,0 +1,50 @@
+Improved Snapshot Testing
+=========================
+
+## Introduction
+
+This proposal is to replace the library that we use for snapshot testing ([`iOSSnapshotTestCase`](https://github.com/uber/ios-snapshot-test-case/)) with [`SnapshotTesting`](https://github.com/pointfreeco/swift-snapshot-testing).
+
+## Motivation
+
+### Cons of `iOSSnapshotTestCase`
+* `iOSSnapshotTestCase` is an old library written in Objective-C that can only snapshot a `UIView` or `CALayer`.
+* It has on occasion seemed to be quite fragile, reporting diffs when nothing changed, to the extent that we have had to increase the failure tolerance to quite a high value, and have even on a couple of occasions considered dropping snapshot testing altogether.
+* it stores all references images in a single folder for all tests
```

Comment:
in fact it can be configured with launch arguments, we just do it like so for some reason

Reply:
Actually Took me six months to have a Pr merged where you can actually set each test folder.